x-pack/test/case_api_integration/basic/tests/cases/sub_cases/find_sub_cases.ts

```diff
@@ -35,227 +48,431 @@ export default ({ getService }: FtrProviderContext): void => {
-          .expect(200);
+        it('should sort on createdAt field in ascending order', async () => {
+          const { newSubCaseInfo: caseInfo } = await createSubCase({ supertest, actionID });
+          // this will result in one closed case and one open
+          await createSubCase({
+            supertest,
+            caseID: caseInfo.id,
+            actionID,
+          });
+
+          const { body }: { body: SubCasesFindResponse } = await supertest
+            .get(`${getSubCasesUrl(caseInfo.id)}/_find?sortField=createdAt&sortOrder=asc`)
+            .expect(200);
+
+          expect(body.total).to.be(2);
+          expect(body.subCases[0].status).to.be(CaseStatuses.closed);
+          expect(body.subCases[1].status).to.be(CaseStatuses.open);
+          expect(body.count_closed_cases).to.be(1);
+          expect(body.count_open_cases).to.be(1);
+          expect(body.count_in_progress_cases).to.be(0);
+        });
 
-        expect(body.total).to.be(1);
-        expect(body.subCases[0].status).to.be(CaseStatuses['in-progress']);
-        expect(body.count_closed_cases).to.be(1);
-        expect(body.count_open_cases).to.be(0);
-        expect(body.count_in_progress_cases).to.be(1);
+        it('should sort on updatedAt field in ascending order', async () => {
+          const { newSubCaseInfo: caseInfo } = await createSubCase({ supertest, actionID });
+          // this will result in one closed case and one open
+          const { newSubCaseInfo: secondSub } = await createSubCase({
+            supertest,
+            caseID: caseInfo.id,
+            actionID,
+          });
+
+          await setStatus({
+            supertest,
+            cases: [
+              {
+                id: secondSub.subCases![0].id,
+                version: secondSub.subCases![0].version,
+                status: CaseStatuses['in-progress'],
+              },
+            ],
+            type: 'sub_case',
+          });
+
+          const { body }: { body: SubCasesFindResponse } = await supertest
+            .get(`${getSubCasesUrl(caseInfo.id)}/_find?sortField=updatedAt&sortOrder=asc`)
+            .expect(200);
+
+          expect(body.total).to.be(2);
+          expect(body.subCases[0].status).to.be(CaseStatuses.closed);
+          expect(body.subCases[1].status).to.be(CaseStatuses['in-progress']);
+          expect(body.count_closed_cases).to.be(1);
+          expect(body.count_open_cases).to.be(0);
+          expect(body.count_in_progress_cases).to.be(1);
+        });
       });
 
-      it('should sort on createdAt field in descending order', async () => {
-        const { newSubCaseInfo: caseInfo } = await createSubCase({ supertest, actionID });
-        // this will result in one closed case and one open
-        await createSubCase({
-          supertest,
-          caseID: caseInfo.id,
-          actionID,
+      describe('pagination', () => {
+        const numCases = 4;
```

Comment:
Only doing 4 sub cases for the pagination because it takes a long time create the sub cases.